crates/holochain/src/conductor/interface.rs

```diff
@@ -31,6 +32,11 @@ impl SignalBroadcaster {
         Ok(())
     }
 
+    /// get the inner vec of senders
+    pub fn into_inner(&self) -> Vec<broadcast::Sender<Signal>> {
+        self.0.clone()
+    }
```

Comment:
without this we can't retrieve the Vec of sender for the sake of attaching new listeners either. so it's very limiting.

Reply:
@Connoropolous do you need individual access to each sender, or does it perhaps make sense to provide a method to listen to all of them at once instead?

Reply:
soo... technically we could leave this bit out for now, since I don't need it. I thought it was funny that there would be no way to start listening, if you have one of these SignalBroadcaster structs...
I don't know what the ultimate needs for this would be

Reply:
Yeah I'd prefer to not give access to the internals if we don't know what the use case will be.